src/app/shared/toast/toast.component.ts

```diff
@@ -36,17 +40,44 @@ export class ToastComponent implements OnInit, OnDestroy {
     this.stringHTML = this.toast.stringHTML
     this.type = this.toast.type
     this.links = this.toast.links
+    this.plainText = this.stringHTML.replace(/<(?:.|\n)*?>/gm, '')
```

Comment:
Let's look at alternatives / pre-built functions available for stripping HTML tags here, when defining the plain text string.

Reply:
although, this is sort of nitpick.. we shouldn't be performing low level functionality (like stripping html tags) at such a high level in our application